SokolTextGame/World.cs

```diff
@@ -11,11 +11,13 @@ private void CommandDictionary()
             commands["who"] = new WhoAmI();
```

Comment:
Please change the name of the `WhoAmI` command to just `Who`. The command names have to be generic as they should be theoretically able to process several user inputs, not just the `who am i` in this case.